docs/api-sdk/SDK_MODELS_HIERARCHY.md

```diff
@@ -20,13 +20,20 @@ The purpose of the revivers is to convert JSON parsed object to the model the re
 Example of a reviver:
 
 ```typescript
-export function reviveMyModel<T extends MyModel = MyModel>(data: any, dictionaries?: any): T  | undefined {
+export function reviveMyModel<T extends MyModelNoDictionary = MyModel>(data: any, dictionaries?: any): T  | undefined {
   if (!data) { return ; }
   data.myField = reviverMyFiedType<MyFiedType>(data.myField, dictionaries);
   return data as T;
 }
 ```
 
+> [!TIP]
+> The reviver can help to remove the dictionary fields from a revived object when used without dictionary:
```

Comment:
So the one without the dictionnary is not the one generated by default?

Reply:
Sorry I am not sure to understand your question :S